avocado_vt/test.py

```diff
@@ -84,17 +84,16 @@ class VirtTest(test.Test):
 
     env_version = utils_env.get_env_version()
 
-    def __init__(self, methodName='runTest', name=None, params=None,
-                 base_logdir=None, job=None, runner_queue=None,
-                 vt_params=None):
+    def __init__(self, **kwargs):
         """
-        :note: methodName, name, base_logdir, job and runner_queue params
+        :note: methodName, name, base_logdir, job/config and runner_queue params
                are inherited from test.Test
         :param params: avocado/multiplexer params stored as
                        `self.avocado_params`.
         :param vt_params: avocado-vt/cartesian_config params stored as
                           `self.params`.
         """
+        vt_params = kwargs.pop("vt_params", None)
```

Comment:
What is the planned difference between `vt_params` and `params_vt`? I think the presence of both could be confusing for people and perhaps it is best to provide a single instance of the `Params` class for all usage within VT.

Reply:
I just follow the naming from the master branch. I agree with you that this can be change, but I think it is not related to this PR. And because I am planing to use this as an example, for other extensions to keep compatibility with Avocado 86.0 and later, to the community, so I would like to make it as simple as possible.

Reply:
Yes no worries, I was suspecting that and I would do the same. I was rather asking if you know more about these and if my comment makes sense to you for a future PR. In any case it is something to consider.